doc/man7/provider-rand.pod

```diff
@@ -110,6 +116,19 @@ OSSL_FUNC_rand_nonce() is used to generate a nonce of the given I<strength> with
 length from I<min_noncelen> to I<max_noncelen>. If the output buffer I<out> is
 NULL, the length of the nonce should be returned.
 
+OSSL_FUNC_rand_get_seed() is used by deterministic generators to
+obtain their seeding material from their parent.  The seed bytes will meet
+the specified
+security level of I<entropy> bits and there will be between I<min_len>
```

Comment:
strange line spacing here.

Reply:
It's a POD file, it doesn't matter :)

I've reformatted the paragraph.